Testing/VelaptorTests/ExtensionMethods/TextBoxExtensionsTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿// <copyright file="TextBoxExtensionsTests.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace VelaptorTests.ExtensionMethods;
+
+using Velaptor.UI;
+
+/// <summary>
+/// Tests all of the extension methods related to the <see cref="TextBox"/> control.
+/// </summary>
+public class TextBoxExtensionsTests
+{
+    // TODO: Create tests
```

Comment:
Is this TODO still valid? 
```suggestion
    // TODO: Create tests
```